extensions/ql-vscode/test/vscode-tests/cli-integration/queries.test.ts

```diff
@@ -136,16 +136,16 @@ describeWithCodeQL()("Queries", () => {
     }
 
     async function runQueryWithExtensions() {
-      const result = new CompletedQueryInfo(
-        await qs.compileAndRunQueryAgainstDatabase(
-          dbItem,
-          await mockInitialQueryInfo(queryUsingExtensionPath),
-          join(tmpDir.name, "mock-storage-path"),
-          progress,
-          token,
-        ),
+      const queryInfo = await qs.compileAndRunQueryAgainstDatabase(
+        dbItem,
+        await mockInitialQueryInfo(queryUsingExtensionPath),
+        join(tmpDir.name, "mock-storage-path"),
+        progress,
+        token,
       );
 
+      const result = new CompletedQueryInfo(queryInfo.query, queryInfo.result);
```

Comment:
I think this will fix the tests. We're not passing in all the necessary optional parameters. I'll also comment on the `CompletedQueryInfo` definition because I think we can help make this error less likely for future developers.
```suggestion
      const result = new CompletedQueryInfo(
        queryInfo.query,
        queryInfo.result,
        queryInfo.logFileLocation,
        queryInfo.successful,
        queryInfo.message,
      );
```